api/v1alpha1/kamajicontrolplane_types.go

```diff
@@ -162,6 +162,7 @@ type KamajiControlPlaneStatus struct {
 
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
+//+kubebuilder:resource:categories=cluster-api;kamaji,shortName=kjcp
```

Comment:
```suggestion
//+kubebuilder:resource:categories=cluster-api;kamaji,shortName=ktcp
```
**K**amaji
**T**enant
**C**ontrol
**P**lane

Any suggestion here is really valuable.